inc/options/general.php

```diff
@@ -323,7 +323,7 @@
 		<p>
 			<?php
 echo sprintf(
-	w3tc_er( 'reverseproxy.general.header', 'A reverse proxy adds scale to an server by handling requests before WordPress does. Purge settings are set on the <a href="%s">Page Cache settings</a> page and <a href="%s">Browser Cache settings</a> are set on the browser cache settings page.' ),
+	w3tc_er( 'reverseproxy.general.header', 'A reverse proxy adds scale to an server by handling requests before WordPress does. Purge settings are set on the <a href="%s">Page Cache settings</a> page and <a href="%s">Browser Cache settings</a> are set on the browser cache settings page.', 'w3-total-cache' ),
```

Comment:
Not a translation function. https://github.com/W3EDGE/w3-total-cache/blob/2.1.5/w3-total-cache-api.php#L601
Use `__()` inside of a `sprintf()` with a `// translators: 1:` comment.

Reply:
changes reverted